buildSrc/build.gradle

```diff
@@ -78,50 +89,80 @@ dependencies {
   compile 'org.apache.rat:apache-rat:0.11'
 }
 
-File tempPropertiesFile = new File(project.buildDir, "version.properties")
-task writeVersionProperties {
-  inputs.properties(props)
-  outputs.file(tempPropertiesFile)
-  doLast {
-    OutputStream stream = Files.newOutputStream(tempPropertiesFile.toPath());
-    try {
-      props.store(stream, "UTF-8");
-    } finally {
-      stream.close();
+
+/*****************************************************************************
+ *                    Bootstrap repositories and IDE setup                   *
+ *****************************************************************************/
+// this will only happen when buildSrc is built on its own during build init
+if (project == rootProject) {
+
+  // TODO: move common IDE configuration to a common file to include
+  apply plugin: 'idea'
+  apply plugin: 'eclipse'
+
+  repositories {
+    mavenCentral()
+    maven {
+      name 'sonatype-snapshots'
+      url "https://oss.sonatype.org/content/repositories/snapshots/"
     }
   }
-}
-
-processResources {
-  dependsOn writeVersionProperties
-  from tempPropertiesFile
-}
 
-extraArchive {
-  javadoc = false
-  tests = false
-}
+  idea {
+    module {
+      inheritOutputDirs = false
+      outputDir = file('build-idea/classes/main')
+      testOutputDir = file('build-idea/classes/test')
+    }
+  }
 
-idea {
-  module {
-    inheritOutputDirs = false
-    outputDir = file('build-idea/classes/main')
-    testOutputDir = file('build-idea/classes/test')
+  eclipse {
```

Comment:
Yes and no. The build-tools project, yes. And I think that is good enough (we can edit in build-tools, and it will be reflected when building buildSrc).